### DISCUSSION TITLE
Which plugin/config handles the UI (pop-up window) when getting import suggestions?

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#1708] [General]
When I create a new instance of a class, and have multiple import options, and one of the options has many constructor params, the UI is totally messed up.

I have a hard time actually seeing where the import is from.
![Screenshot 2023-10-14 at 00 50 08](https://github.com/LazyVim/LazyVim/assets/59118341/8c20a38c-3701-4937-8e5b-e64d3e514046)

What is the name of this plugin so I can make some UI changes that makes it easier to see where the import is coming from?

**Comment by lkhphuc** [comment]
It's nvim-cmp, the completion plugin. https://github.com/hrsh7th/nvim-cmp/wiki/Menu-Appearance
